examples/viewer.html

```diff
@@ -343,11 +395,30 @@
     }
 
 
-    var viewer;
-
     function loadAmpDoc() {
-      viewer = new Viewer();
-      viewer.start();
+      new Viewer('container1', '1', true).start();
+      addShowContainer('1');
+
+      new Viewer('container2', '2', false).start();
+      addShowContainer('2');
+
+      showContainer('1');
+    }
+
+    function addShowContainer(id) {
+      var anchor = document.getElementById('show-container' + id + '-anchor');
+      anchor.onclick = function() {
+        showContainer(id);
+      };
+    }
+
+    function showContainer(id) {
+      log('Show container: ', id);
+      var viewerEl = document.querySelector('viewer');
+      viewerEl.setAttribute('data-show-container', id);
```

Comment:
Only for convenience. Easier to set a single value than toggle different classes on/off.